docs/workloads/aws/wordpress-aws.md

```diff
@@ -148,79 +161,110 @@ It will take a while (~15 minutes) for the EKS cluster to be deployed and become
+$ KUBECONFIG=/tmp/$CLUSTER kubectl get pods,services,deployments -A
+NAMESPACE     NAME                             READY   STATUS                       RESTARTS   AGE
+kube-system   pod/aws-node-d2sv2               1/1     Running                      0          58m
+kube-system   pod/coredns-7b5c8bfcfc-8cknr     1/1     Running                      0          85m
+kube-system   pod/coredns-7b5c8bfcfc-nsckc     1/1     Running                      0          85m
+kube-system   pod/kube-proxy-bxrkh             1/1     Running                      0          58m
+wordpress     pod/wordpress-8545774bcf-ppr4l   0/1     CreateContainerConfigError   0          44m
+
+NAMESPACE     NAME                 TYPE           CLUSTER-IP       EXTERNAL-IP                                                               PORT(S)         AGE
+default       service/kubernetes   ClusterIP      10.100.0.1       <none>                                                                    443/TCP         85m
+kube-system   service/kube-dns     ClusterIP      10.100.0.10      <none>                                                                    53/UDP,53/TCP   85m
+wordpress     service/wordpress    LoadBalancer   10.100.122.109   ae429ee31969e11e9b7eb06064878ead-1157393490.eu-west-1.elb.amazonaws.com   80:30297/TCP    44m
+
+NAMESPACE     NAME                              DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+kube-system   deployment.extensions/coredns     2         2         2            2           85m
+wordpress     deployment.extensions/wordpress   1         1         1            0           44m
+```
+
+Continuing as the application developer, we can watch the WordPress pod come online and a public IP address will get assigned to it:
+
+```console
+kubectl -n complex get kubernetesapplication,kubernetesapplicationresources
 ```
 
 When a public IP address has been assigned, you'll see output similar to the following:
 
 ```console
-NAME            CLUSTER        NAMESPACE   DEPLOYMENT   SERVICE-EXTERNAL-IP
-demo            demo-cluster   demo        wordpress    104.43.240.15
+NAME                                                          CLUSTER                  STATUS               DESIRED   SUBMITTED
+kubernetesapplication.workload.crossplane.io/wordpress-demo   wordpress-demo-cluster   PartiallySubmitted   3         1
+
+NAME                                                                             TEMPLATE-KIND   TEMPLATE-NAME   CLUSTER                  STATUS
+kubernetesapplicationresource.workload.crossplane.io/wordpress-demo-deployment   Deployment      wordpress       wordpress-demo-cluster   Failed
+kubernetesapplicationresource.workload.crossplane.io/wordpress-demo-namespace    Namespace       wordpress       wordpress-demo-cluster   Submitted
+kubernetesapplicationresource.workload.crossplane.io/wordpress-demo-service      Service         wordpress       wordpress-demo-cluster   Failed
```

Comment:
I'm drafting this PR while I am encountering some hiccups.  I'll push over this when the instructions provide a hiccup free run through.

Reply:
resolved by 0d719b3f2a4ad96ac5c6068526ed09a71de8b84b